multus/multus.go

```diff
@@ -190,6 +193,16 @@ func cmdAdd(args *skel.CmdArgs, exec invoke.Exec, kubeClient k8s.KubeClient) (cn
 		if delegate.MasterPlugin || result == nil {
 			result = tmpResult
 		}
+
+		//create the network status, only in case Multus as kubeconfig
+		if n.Kubeconfig != "" {
+			delegateNetStatus, err := types.LoadNetworkStatus(tmpResult, delegate.Name, delegate.MasterPlugin)
+			if err != nil {
+				return nil, fmt.Errorf("Multus: Err rethe networks status: %v", err)
```

Comment:
Not quite sure what you meant with "rethe" here; maybe "Multus: error loading the network status"?